### PR TITLE
Govpress g9

### PR DESCRIPTION
**Pull request by harry-m**
Contract updates for G9, including SLA.

### CODE REVIEW

File: agreements/service_level_agreement.md [hunk 1]
```diff
@@ -0,0 +1,96 @@
...
+
+    3. Clients are requested to describe the problem and its impact as clearly
+       as possible, in order to allow us to set priority appropriately.
+
+    4. Tickets are set to Normal priority as a matter of course on receipt.
+
+    5. Tickets sent to support-emergency@dxw.com are set to Urgent priority on
+       receipt.
+
+    6. The receipt of an Urgent ticket will result in the immediate alerting of
```
Comment (dgmstuart) [inline agreements/service_level_agreement.md]: Do 6,7,8 belong in the next section? Or possibly grouped together in some other way?

File: agreements/service_level_agreement.md [hunk 2]
```diff
@@ -0,0 +1,96 @@
...
+          completely unavailable, or whose performance is so poor as to render
+          the service unusable;
+       2. A production system may have been compromised by an attacker and that
+          the suspected attack or its impact is ongoing.
+
+    2. Urgent tickets must receive a first reply within 1 hour (standard) or 30
+       minutes (enhanced), at any time of the day or night.
+
+    3. Tickets created by emailing emergency-support@dxw.com are automatically
+       marked as urgent.
```
Comment (dgmstuart) [inline agreements/service_level_agreement.md]: This is repeated from above - is that for legal-y reasons?

File: agreements/service_level_agreement.md [hunk 3]
```diff
@@ -0,0 +1,96 @@
...
+       emergency, as defined below.
+
+    8. Use of the emergency support address may be withdrawn from clients who
+       use it to submit tickets in situations that do not constitute an emergency.
+
+    9. References to "working" days, hours or minutes refer to dxw's working
+       hours of 10:00-18:00 Monday-Friday, excluding bank holidays.
+
+    10. The "enhanced" response times apply to tickets raised by customers on our
+        Managed Plus plan.
```
Comment (dgmstuart) [inline agreements/service_level_agreement.md]: 9 and 10 are defining terms - would it be more readable to have them in their own section?

Comment (harry-m) [inline agreements/service_level_agreement.md]: I think there aren't enough definitions of things for that.

File: agreements/service_level_agreement.md (hunk 1 above)
Comment (harry-m) [inline agreements/service_level_agreement.md]: No, I don't think so - the subsequent sections are focussed on the definitions of the respective priorities and our target response times.

File: agreements/service_level_agreement.md (hunk 3 above)
Comment (harry-m) [inline agreements/service_level_agreement.md]: I think that would make it less readable - don't think there are enough terms being defined to need their own section.

File: agreements/service_level_agreement.md (hunk 2 above)
Comment (harry-m) [inline agreements/service_level_agreement.md]: Mostly just to reinforce it, as it's an important point that is often overlooked.

**Pull request merged**